package.json

```diff
@@ -13,8 +13,8 @@
   "scripts": {
     "lint": "eslint ./",
     "test": "npm run lint && mocha tests --recursive",
-    "test-single": "mocha tests/lib/rules/mapStateToProps-prefer-parameters-names",
-    "debug-test": "mocha --debug-brk --inspect tests/lib/rules/mapStateToProps-prefer-parameters-names",
+    "test-single": "mocha tests/lib/rules/mapStateToProps-no-store",
```

Comment:
it seems strange that "test-single" would hardcode a specific test here; is this meant to be like `npm run test-single path/to/file` instead?

Reply:
yes.

Reply:
you could change this to just `mocha`, and then it would work that way :-)

Reply:
uhm, yes let me just remove these